test/aerospike_clj/key_test.clj

```diff
@@ -0,0 +1,57 @@
+(ns aerospike-clj.key-test
+  (:require [aerospike-clj.key :refer [create-key]]
+            [clojure.test :refer [deftest is]]
+            [taoensso.timbre :refer [spy]]
+            [clojure.string :as s])
+  (:import [com.aerospike.client Key Value]
+           [com.aerospike.client.util ThreadLocalData]))
+
+(defn- compare-keys 
+  ([_k1] true)
+  ([k1 k2] (= (spy (seq (.digest ^Key k1)))
+              (spy (seq (.digest ^Key k2)))))
+  ([^Key k1 ^Key k2 & more]
+   (if (= (seq (.digest ^Key k1))
+          (seq (.digest ^Key k2)))
+     (if (next more)
+       (recur k2 (first more) (next more))
+       (compare-keys k2 (first more)))
+     false)))
+      
+(deftest create-with-digest
+  (let [as-ns "ns"
+        as-set "set"
+        as-key "key" 
+        k (create-key as-key as-ns as-set)
+        k-digest (.digest k)]
+    (is (.equals k 
+                 (create-key k-digest as-ns as-set (Value/get "user-key"))))))
+
+(deftest create-with-digest-and-byte-array
+  (let [as-ns "ns"
+        as-set "set"
+        ba (byte-array 10)]
+      (doseq [i (range 10)]
+        (aset-byte ba i i))
+      (let [k (create-key ba as-ns as-set nil)
+            k-digest (.digest k)]
+        (is (.equals k 
+                     (create-key k-digest as-ns as-set nil))))))
+
```

Comment:
I think we're missing a key created from `Value` test.
create-key that is used on #28 (Value/get) is a string.

Reply:
`(Value/get)` returns a `Value` not `String`.